src/database/network-table.c

```diff
@@ -2144,6 +2144,85 @@ char *__attribute__((malloc)) getNameFromIP(sqlite3 *db, const char *ipaddr)
 	return name;
 }
 
+// Get most recently seen host name of device identified by MAC address
+char *__attribute__((malloc)) getNameFromMAC(const char *client)
+{
+	// Return early if database is known to be broken
+	if(FTLDBerror())
+		return NULL;
+
+	log_info("Looking up host name for %s", client);
```

Comment:
I'm a big fan of logging, but this is too much. It logs every time the client page is loaded

```
2023-11-21 21:01:55.298 [16140/T16149] INFO: Looking up host name for 3C:97:0E:13:36:C0
2023-11-21 21:01:55.307 [16140/T16149] INFO: Looking up host name for 80:86:F2:9C:57:16
2023-11-21 21:02:14.773 [16140/T16149] INFO: Looking up host name for 3C:97:0E:13:36:C0
2023-11-21 21:02:14.775 [16140/T16149] INFO: Looking up host name for 80:86:F2:9C:57:16
2023-11-21 21:04:04.046 [16140/T16153] INFO: Looking up host name for 3C:97:0E:13:36:C0
2023-11-21 21:04:04.054 [16140/T16153] INFO: Looking up host name for 80:86:F2:9C:57:16
2023-11-21 21:04:10.233 [16140/T16154] INFO: Looking up host name for 3C:97:0E:13:36:C0
2023-11-21 21:04:10.235 [16140/T16154] INFO: Looking up host name for 80:86:F2:9C:57:16
2023-11-21 21:04:13.450 [16140/T16148] INFO: Looking up host name for 3C:97:0E:13:36:C0
2023-11-21 21:04:13.456 [16140/T16148] INFO: Looking up host name for 80:86:F2:9C:57:16

```

```suggestion
	log_info("Looking up host name for %s", client);
```
```suggestion
	log_debug(DEBUG_DATABASE,"Looking up host name for %s", client);
```